vispy/scene/widgets/grid.py

```diff
@@ -496,6 +492,7 @@ def _update_child_widget_dim(self):
                 y = np.sum(value_vectorized(self._height_grid[col][0:row]))
 
             if isinstance(widget, ViewBox):
+                print("####", val, x, y, width, height)
```

Comment:
Lets remove this print statement.

Reply:
That's what I get for making a PR right before bed. Done.